README.md

```diff
@@ -45,6 +45,18 @@ Examples:
   bitmask.to_i.to_s(2)    # => "101"
 
   Bitmask.new(masks, 5) == Bitmask.new(masks, { :cat => true, :dog => false, :fish => true }) # => true
+
+  # Least Significant Bit - LSB
+  bitmask = Bitmask.new(masks, {:cat => true, :fish => true})
+  bitmask.lsb # => 1
+  bitmask.lsb.to_s(2) # => "1"
+  Bitmask.new(masks, bitmask.lsb).to_a # => [:cat]
+
+  # Most Significant Bit - MSB
```

Comment:
I see that I'm a little late, but it may be worth adding a bit of context for why this is useful, maybe something like what [you wrote here](https://github.com/amiel/bitmask/issues/5#issuecomment-1149081905).

Reply:
Yeah, good idea.  I'll add more documentation.

Reply:
Thanks!